_posts/2020-04-20-origami-survey-results.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Origami Survey Results, Building Components
+description: TODO
```

Comment:
Missing a description. Figured I'd comment about it in case it gets missed.

Reply:
Oh, good spot!